workflow101.Rmd

```diff
@@ -234,6 +234,13 @@ To see the most useful keyboard shortcuts, press `Alt + Shift + K` or use *Help
 ::: callout-tip
 ## RStudio
 
+RStudio also provides the [Command Palette](https://docs.posit.co/ide/user/ide/reference/shortcuts.html#command-palette) which gives fast, searchable access to all of the IDE's commands -- especially helpful when you can't remember a particular keyboard shortcut.
+It is invoked via `Ctrl+Shift+P` (Cmd+`Shift+P` on macOS).
```

Comment:
Can you check this presentation of keyboard shortcuts for consistency with similar elsewhere?

Reply:
Good catch - they're not backticked in the rest of the book.